modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/internal/security/permission/contributor/test/DepotRoleContributorTest.java

```diff
@@ -54,6 +57,31 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testCMSConsumerRoleAssignment() throws Exception {
+		Group group = GroupTestUtil.addGroup(
+			_group.getCompanyId(), TestPropsValues.getUserId(),
+			GroupConstants.DEFAULT_PARENT_GROUP_ID, GroupConstants.CMS);
+
+		long groupId = group.getGroupId();
+
+		DepotTestUtil.withSiteMember(
+			group,
+			user -> {
+				PermissionChecker permissionChecker =
+					_permissionCheckerFactory.create(user);
+
+				long[] roleIds = permissionChecker.getRoleIds(
+					user.getUserId(), groupId);
```

Comment:
Can groupId be inlined here? (it is only used once)

Reply:
done